README.rst

```diff
@@ -178,6 +178,14 @@ Contributing to DALI
 
 Contributions to DALI are more than welcome. To make the pull request process smooth, please follow these `guidelines <CONTRIBUTING.md>`_.
 
+Reporting problem, asking questions
+-----------------------------------
+
+We appreciate any feedback, question or bug reported regarding this project. To make the process of answering to all those requests smoother please stick to following the guide (it is based on https://stackoverflow.com/help/mcve) when help with code is needed. So posted examples should be:
```

Comment:
"to following the guide" -> "to the following guide"

Reply:
Done